pages/common/docker-top.md

```diff
@@ -0,0 +1,12 @@
+# docker top
+
+> Display the running processes of a container.
+> More information: <https://docs.docker.com/engine/reference/commandline/top>.
+
+- Show help:
+
+`docker top`
+
+- Display the running processes of a container:
+
+`docker top {{container}} {{[ps OPTIONS]}}`
```

Comment:
```suggestion
`docker top {{container}} {{ps OPTIONS}}`
```

What exactly is ps OPTIONS? Would it be possible to add a concrete example here?

Reply:
I tested locally and the command didn't work when passed with ps.

So it should be

```suggestion
`docker top {{container}}`
```